skeleton-typescript-webpack/src/app.html

```diff
@@ -1,7 +1,7 @@
 <template>
   <require from="./nav-bar.html"></require>
   
-  <nav-bar router.bind="router"></nav-bar>
+  <nav-bar router.bind="router" containerless></nav-bar>
```

Comment:
This was used with version 4.0.0 otherwise responsive menu wasn't working. They must have fixed it in 4.1.0. Already discussed in the PR itself, just leaving this note here just so people know possible issues with using 4.0.0.